mdn/README.md

```diff
@@ -26,4 +26,125 @@ As the `mdn-downloads` bucket already exists and was created outside of Terrafor
 terraform import aws_s3_bucket.mdn-downloads mdn-downloads
 ```
 
-This resource is stored in the Terraform S3 state, so the command doesn't need to be run again.
\ No newline at end of file
+This resource is stored in the Terraform S3 state, so the command doesn't need to be run again.
+
+
+## Restoring a database to a persistent MySQL container
+
+- install kubectl and helm, if necessary
+  - this is done as part of the [stage2.sh](../k8s/install/stage2.sh) install or you can manually run the following:
+
+```sh
+source ../k8s/install/stage2_functions.sh
```

Comment:
might be best to set `KOPS_INSTALLER` and source a `config.sh` as well. I believe some of the stage2 functions require the path to KOPS_INSTALLER.